tests/test_modeling_rag.py

```diff
@@ -535,7 +535,6 @@ def config_and_inputs(self):
             n_docs=self.n_docs,
             retrieval_vector_size=self.retrieval_vector_size,
             max_combined_length=self.max_combined_length,
-            use_cache=False,
```

Comment:
use cache was not tested because there was a discrepancy previously between Bart and T5 -> should work now

Reply:
now use_cache is correctly tested in RAG